From cfecebf4241f9a2c4bcd6bfe35ec3fbe89bc618a Mon Sep 17 00:00:00 2001
From: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
Date: Fri, 31 Mar 2023 15:06:16 -0300
Subject: [PATCH] csi-unity: add apiVersion

This patch consists of adding apiVersion on Chart.yaml required by helm lint.

Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Signed-off-by: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
---
 helm/csi-unity/Chart.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/helm/csi-unity/Chart.yaml b/helm/csi-unity/Chart.yaml
index eddbb7a..a2f8d8e 100644
--- a/helm/csi-unity/Chart.yaml
+++ b/helm/csi-unity/Chart.yaml
@@ -1,6 +1,7 @@
 name: csi-unity
 version: 2.6.0
 appVersion: 2.6.0
+apiVersion: v2
 kubeVersion: ">= 1.24.0 < 1.27.0"
 # If you are using a complex K8s version like "v1.24.3-mirantis-1", use this kubeVersion check instead
 # WARNING: this version of the check will allow the use of alpha and beta versions, which is NOT SUPPORTED
--
2.25.1
